app/scripts/migrations/048.js

```diff
@@ -34,5 +36,20 @@ function transformState (state = {}) {
     delete state.NetworkController?.provider?.rpcTarget
   }
 
+  // 3.  Add localhost network to frequentRpcListDetail.
+  if (!state.PreferencesController) {
+    state.PreferencesController = {}
+  }
```

Comment:
```suggestion
  state.PreferencesController ??= {}
```

Reply:
For posterity: this wasn't done because we don't have nullish assignment yet.